runit-python-bionic/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM socrata/runit-bionic
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install python python-dev python-pip
```

Comment:
We may also want to build python from source on these container's as well.

Reply:
Yeah, as services move to bionic the owners will request different versions of python that will require building python from source.